x-pack/plugins/observability_solution/slo/server/services/summay_transform_manager.ts

```diff
@@ -31,6 +31,9 @@ export class DefaultSummaryTransformManager implements TransformManager {
           logger: this.logger,
         }
       );
+      if (startTransform) {
+        await this.start(transformParams.transform_id);
+      }
```

Comment:
If we go with this async change, should we not make that install method always install and start? I don't think there is a place somewhere (should double check) where we use install without start. 

If my statement abobe is true, we could remove the boolean and always start.